### ISSUE TITLE
Design the input and output of forms data

### ISSUE THREAD

**Issue by kdmiller45** [formtools/core#903]
does FormTools have the ability for me to design the look of the input form and report form output.
that is custom place the fields like you can with M.S. access

**Comment by benkeen** [comment]
Yes absolutely. With external and Form Builder forms you can customize how the form looks.
https://docs.formtools.org/userdoc/form_management/form_types/
